src/integration-tests/integration-test.py

```diff
@@ -42,6 +42,7 @@
 class TemplateType(Enum):
     libfuzzer = "libfuzzer"
     libfuzzer_dotnet = "libfuzzer_dotnet"
+    libfuzzer_qemu = "libfuzzer_qemu"
```

Comment:
Would it be more consistent to use a more explicit variant name like:
```suggestion
    libfuzzer_qemu_user = "libfuzzer_qemu_user"
```